packages/bff/src/model/api/tenantApiConverter.ts

```diff
@@ -111,3 +111,52 @@ export function toBffApiVerifiedTenantAttributes(
     )
     .filter(isDefined);
 }
+
+function toBffApiTenantMail(
+  tenantMails: tenantApi.Tenant["mails"]
+): bffApi.Mail | undefined {
+  const mail = tenantMails.find(
+    (m) => m.kind === tenantApi.MailKind.Values.CONTACT_EMAIL
+  );
+  return mail
+    ? {
+        address: mail.address,
+        description: mail.description,
+      }
+    : undefined;
+}
+
+export function toBffApiTenant(
+  tenant: tenantApi.Tenant,
+  certifiedAttributes: tenantApi.CertifiedTenantAttribute[],
+  declaredAttributes: tenantApi.DeclaredTenantAttribute[],
+  verifiedAttributes: tenantApi.VerifiedTenantAttribute[],
+  registryAttributesMap: RegistryAttributesMap
+): bffApi.Tenant {
+  return {
+    id: tenant.id,
+    selfcareId: tenant.selfcareId,
+    externalId: tenant.externalId,
+    createdAt: tenant.createdAt,
+    updatedAt: tenant.updatedAt,
+    name: tenant.name,
+    features: tenant.features,
+    onboardedAt: tenant.onboardedAt,
+    subUnitType: tenant.subUnitType,
```

Comment:
Waiting for https://github.com/pagopa/interop-be-monorepo/pull/936 to be merged so that we have these two fields in the `tenantApi.Tenant`